src/main/java/wooteco/subway/line/domain/Distance.java

```diff
@@ -0,0 +1,47 @@
+package wooteco.subway.line.domain;
+
+import wooteco.subway.exception.InternalLogicException;
+import wooteco.subway.exception.addition.LineDistanceException;
+
+public class Distance {
+    private static final int DEFAULT_FARE_RANGE = 10;
+    private static final int FIRST_FARE_RANGE_DISTANCE = 50;
+    public static final int DEFAULT_TO_FIRST_RANGE_DISTANCE = FIRST_FARE_RANGE_DISTANCE - DEFAULT_FARE_RANGE;
+    private static final int MIN_POSITIVE = 1;
+    private final int distance;
```

Comment:
```suggestion
    private static final int MIN_POSITIVE = 1;
    
    private final int distance;
```

인스턴스 변수와 클래스 변수 사이에 공백을 추가한다면 가독성이 상승합니다 :)

Reply:
오! 수정했어요! :) 감사합니다.